src/main/java/com/emc/pravega/perf/PravegaPerfTest.java

```diff
@@ -70,17 +68,16 @@
     private static int producerCount = 0;
     private static int consumerCount = 0;
     private static int segmentCount = 0;
-    private static int eventsPerSec = 40;
-    private static int runtimeSec = 10;
+    private static int eventsPerWorker = 40;
```

Comment:
i have changed this to 3000 by default;
idea here is , if the user has not supplied for how long he wants to execute the benchmarking then it will run for 24hrs or  writes/reads 3000 events by one producer/reader which ever is earlier/less.